libmamba/src/core/shell_init.cpp

```diff
@@ -382,6 +391,30 @@ namespace mamba
         return content.str();
     }
 
+    std::string
+    nu_content(const fs::u8path& env_prefix, const std::string& /*shell*/, const fs::u8path& mamba_exe)
+    {
+        std::stringstream content;
+        std::string s_mamba_exe;
+        if (on_win)
+        {
+            s_mamba_exe = native_path_to_unix(mamba_exe.string());
+        }
+        else
+        {
+            s_mamba_exe = mamba_exe.string();
+        }
+
+        content << "\n# >>> mamba initialize >>>\n";
+        content << "# !! Contents within this block are managed by 'mamba init' !!\n";
+        content << "$env.MAMBA_EXE = " << mamba_exe << "\n";
+        content << "$env.MAMBA_ROOT_PREFIX = " << env_prefix << "\n";
+        content << "$env.PATH = ($env.PATH | prepend ([$env.MAMBA_ROOT_PREFIX bin] | path join) | uniq)\n";
+        content << "source " << env_prefix.string() << "/etc/profile.d/micromamba.nu\n";
```

Comment:
I don't like this; we are planning to get rid of the `profile.d` folder entirely in the future.